src/test/java/org/takes/tk/TkProxyTest.java

```diff
@@ -49,23 +50,273 @@
      */
     @Test
     public void justWorks() throws Exception {
-        new FtRemote(new TkFixed("hello, world!")).exec(
+        final String testText = "Hello, world!";
+        new FtRemote(new TkFixed(testText)).exec(
             new FtRemote.Script() {
                 @Override
                 public void exec(final URI home) throws IOException {
                     MatcherAssert.assertThat(
                         new RsPrint(
                             new TkProxy(
-                                String.format(
-                                    "%s:%d",
-                                    home.getHost(), home.getPort()
-                                )
+                                TkProxyTest.this.formatProxyHost(home)
                             ).act(new RqFake())
                         ).print(),
-                        Matchers.containsString("hello")
+                        Matchers.containsString(testText)
+                    );
+                }
+            }
+        );
+    }
+
+    /**
+     * TkProxy can work with GET HTTP method
+     * Plus 4 random headers checked.
+     * @throws Exception If some problem inside
+     */
+    @Test
+    public void actsOnHttpGETWithRequestHeaders() throws Exception {
```

Comment:
@aschworer `GET` is not an abbreviation. Please, use camel case
```actsOnHttpGetWithRequestHeaders```
Same related to the other test methods.
Also, usually, we place a full stop in the end of the first sentence of javadoc.
http://www.oracle.com/technetwork/java/javase/documentation/index-137868.html#firstsentence